Documentation/admin_guide.md

```diff
@@ -65,7 +60,7 @@ cluster is healthy
 
 #### Runtime Metrics
 
-etcd uses [Prometheus](http://prometheus.io/) for metrics reporting in the server. You can read more through the runtime metrics [doc](metrics.md).
+etcd uses [Prometheus] for metrics reporting in the server. You can read more through the runtime metrics [doc][metrics].
```

Comment:
While it is quasi-legal Markdown to have these empty link declarations (`[Prometheus]`), it will be easier to read and save us trouble with different markdown interpreters in future if we write explicit declarations for every link: 
`[Prometheus][prometheus]`